### PR TITLE
Admin tags and categories

### PR DESCRIPTION
**Pull request by appinteractive**
**Use with https://github.com/Human-Connection/Nitro-Backend/pull/62**

Added Tags and Categories to the Admin area

grafik
grafik

### CODE REVIEW

**Comment by appinteractive** [comment]
Then the docker pipeline is not working correctly. Locally and on Travis it works correctly. @roschaefer for that you are currently the expert. Any suggestions?

**Comment by appinteractive** [comment]
As I do not use pc as my driver I would be way more then 600€. So I can’t afford it at the moment. And as docker is frying my seven year old machine. So I will use the powers of my native system for now 😉

**Comment by roschaefer** [comment]
@appinteractive I see the above bug on `master` now. What has happened?

**Comment by roschaefer** [comment]
I am finally able to build the docker container but just don't know what I have changed. I believe, we have some .gitignored folders in the styleguide that can cause trouble when you switch back and forth branches.

**Comment by roschaefer** [comment]
I merged in https://github.com/Human-Connection/Nitro-Web/pull/84 consider to merge this PR before reviewing this PR.

File: layouts/default.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
               style="float: right"
             >
               <a
+                class="avatar-menu"
```
Comment (roschaefer) [inline layouts/default.vue]: @appinteractive do we use some sort of fancy CSS naming like BEM?

Comment (appinteractive) [reply layouts/default.vue]: Yeah kinda, as you can see we prefix the styles with the component name. But not actually in the ugly BEM style.

**Comment by appinteractive** [comment]
@roschaefer the wrong backend branch is checked out! That’s why it failes.

**Comment by roschaefer** [comment]
@appinteractive ha! The PR check fails because it's on `master`. How do we tackle that? :thinking:

**Comment by roschaefer** [comment]
@appinteractive maybe we can start with a workflow to merge the backend first and then run the full stack tests again?

**Comment by roschaefer** [comment]
@appinteractive green! Wohoo

**Pull request merged**